.github/workflows/generate-docs.yml

```diff
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Generate Automatic Documentation
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push
+  push:
+    branches:
+      - actions-dev
+      - develop
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
```

Comment:
you can remove this kind of comments that explains how workflow works. Just keep comments that make sens.

Reply:
Yep, I'll do this ASAP.

Reply:
Yep, I added a name to each step so that it's easier to understand.